tools/lsp/common.rs

```diff
@@ -74,3 +76,36 @@ pub enum PreviewToLspMessage {
     PreviewTypeChanged { is_external: bool },
     RequestState { unused: bool }, // send all documents!
 }
+
+/// Information on the Element types available
+#[derive(Clone, Debug, serde::Deserialize, serde::Serialize)]
+pub struct ComponentInformation {
```

Comment:
Is it Component or Element ?

Reply:
```
component Button { // <-- Component
}

export component App {
    VerticalLayout {
        Button { // <-- Element
            text: "Launch";
        }
    }
}
```

Since here we're talking about the former, not the latter, I'd say `ComponentInformation` is the correct name. But the comment above as well as the description of the PR seems wrong to me.